week-1-practise/level-2/3-currency-conversion.js

```diff
@@ -5,7 +5,9 @@
   Write a function that converts a price to USD (exchange rate is 1.4 $ to £)
 */
 
-function convertToUSD() {}
+function convertToUSD(pound) {
+  return pound*1.4;
+}
 
 /* DO NOT EDIT BELOW THIS LINE */
 /* --------------------------- */
```

Comment:
As per the comment on the previous question, what happens if this function is being used to convert currencies other than "pound" to $? Would the way the function is written still make sense, particularly when read by another programmer at a later date?